accepted/2021/utf8/README.md

```diff
@@ -0,0 +1,21 @@
+# Improve UTF-8 support
+
+**DRAFT**
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+UTF-8 is currently the most popular encoding on the web with [more than 95% of all websites using it](https://w3techs.com/technologies/cross/character_encoding/ranking). Additionally it's used as a default encoding on Linux. When .NET was created there was many character encodings used widely and UTF-8 was not dominant at the time - .NET has chose to use UTF-16. As UTF-8 became more popular it become more obvious that UTF-8 has many memory advantages over other encodings including UTF-16 and therefore its great adoption.
```

Comment:
In addition to Linux, UTF-8 is also going to become the preferred code page on Windows in a future release.

Reply:
I don't believe that's much of an all-in: the -A APIs internally convert to UTF-16 and then call the appropriate -W API. This is just increasing compatibility with many programs which blindly assumed that -A APIs had UTF-8. I don't expect the native UTF-16 encoding to ever flip.

Reply:
I agree that the Windows OS as a whole is always going to be UTF-16 native. I guess that there may be clusters of new APIs that are UTF-8 native.

Reply:
> I guess that there may be clusters of new APIs that are UTF-8 native.

This matches my understanding.

Reply:
I'll leave this as is since I'm not seeing any definite source of truth for what Windows is going to do. The only thing we can say for sure is that they recommend UTF-8 usage